helm_deploy/laa-estimate-eligibility/templates/_envs.tpl

```diff
@@ -130,5 +130,12 @@ env:
         key: GOOGLE_OAUTH_CLIENT_SECRET
   - name: SEED_ADMINS
     value: {{ .Values.app.seedAdmins }}
+  {{ if .Values.metabase.enabled }}
+  - name: MB_DB_CONNECTION_URI
```

Comment:
As I understand it, putting this here will make the MD_DB_CONNECTION_URI available as an env var in the Rails application pods. Do we need it there? You've got it set up as an env var in metabase-deployment.yaml. Isn't that the only place it needs to be?